node/src/BaseClient.ts

```diff
@@ -1224,6 +1225,33 @@ export class BaseClient {
         );
     }
 
+    /** Moves `member` from the set at `source` to the set at `destination`, removing it from the source set.
+     * Creates a new destination set if needed. The operation is atomic.
+     * See https://valkey.io/commands/smove for more details.
+     *
+     * Note: When in cluster mode, `source` and `destination` must map to the same hash slot.
+     *
+     * @param source - The key of the set to remove the element from.
+     * @param destination - The key of the set to add the element to.
+     * @param member - The set element to move.
+     * @returns True on success, or False if the `source` set does not exist or the element is not a member of the source set.
+     *
+     * @example
+     * ```typescript
+     * const result = await client.smove("set1", "set2", "member1");
+     * console.log(result); // Output: True - "member1" was moved from "set1" to "set2".
```

Comment:
true/false are lowercase in node
```suggestion
     * console.log(result); // Output: true - "member1" was moved from "set1" to "set2".
```